ui/src/MainPage.tsx

```diff
@@ -23,7 +23,7 @@ export function MainPage(): JSX.Element {
 
   return (
     <>
-      <Title style={{ textAlign: "center" }}>No Waste</Title>
+      <Title style={{ textAlign: "center" }}>{t("mainPage.title")}</Title>
```

Comment:
Do we want to translate the name of the app? No one translates Facebook as "Twarzo-ksiazka" for example ;p

@jaworskak Any thoughts?

Reply:
I corrected